docs/website/views/index.11tydata.yml

```
@@ -10,20 +10,22 @@ landing:
     description: >
       Make AEM project setup simple and quick to market with our component-oriented
       open-source Exadel Authoring Kit for Adobe Experience Manager
-
-
+    buttons:
+      - link: content/getting-started/installation
+        text: Get started
+      - link: https://github.com/exadel-inc/etoolbox-authoring-kit
+        text: Check it out
 
   features:
     title: What’s the Exadel Authoring Kit for AEM all about?
     items:
-      - A smart thoroughly tested extension for AEM authoring interfaces
       - Meant for developers, authors, and business analysts working in an AEM environment
-      - Filled with reusable coding and visual patterns
       - Rich in utilities for handling AEM Sling models, modifying AEM component policy,
-        and managing structured data
+          and managing structured data
```

```suggestion
        and managing structured data
```
Unnecessary whitespace